Packs/Kafka/Integrations/KafkaV3/KafkaV3_description.md

```diff
@@ -15,3 +15,15 @@
 5. Provide the client certificate in the 'Client certificate (.cer)' section.
 6. Provide the client certificate key in the 'Client certificate key (.key)' section.
 7. If your client certificate is password protected, provide the password in the 'Client certificate key password (if required)' section.
+
+## Configuration of SASL_SSL PLAIN:
+1. Make sure you have the broker port '<port>' which supports SSL connection.
+2. Add '<broker_address>:<port>' to the brokers list.
+3. Provide the CA root certificate in the 'CA certificate of Kafka server (.cer)' section.
+4. If your client certificate is password protected, provide the password in the 'Client certificate key password (if required)' section.
+5. Provide SASL PLAIN Username and SASL PLAIN Password
+
+Note: SASL is supported only when used in combination with SSL.
+
+An important point:
```

Comment:
Maybe a different text instead of An important point:?
What ever you think

Reply:
Let's add it to the note, otherwise just say "Important"